.github/workflows/deploy_tagged.yaml

```diff
@@ -61,14 +34,25 @@ jobs:
           tar -zxf git-secrets-1.3.0.tar.gz
           cd git-secrets-1.3.0
           sudo make install   
+
       - name: Set up JDK
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
-          java-version: '17.0.4+8'
+          java-version: '21.0.2+13.0.LTS'
           distribution: 'adopt'
 
+      - name: Set changelist
```

Comment:
This allows us to call the tagged release workflow without providing the changelist version. This info is already part of the tag so this step is just figuring out the changelist version from the tag

Reply:
Neat!
Just to think it through, does this need error handling in the case that someone accidentally makes a tag like 
`1.17-alpha.0`

Reply:
To be clear, not because I want to support it, just to have it die gracefully